content/articles/deploy-django-with-heroku/index.md

```diff
@@ -0,0 +1,478 @@
+![hero](/how-to-deploy-django-project-with-database-using-heroku-cloud-hoisting/django-heroku.png)
+
+### Introduction 
+
+Welcome folks! I am Arafat Olayiwola, a software developer and technical writer. In this tutorial I will be walking you through steps of deploying Django application with relational database using Heroku server by building a `studentprofile` application. Heroku is one of the cloud providers out there that allows developers to hoist their application on their server, and helps by monitoring, managing the health checks of the server. 
+
+To deploy Django application on Heroku server, some dependencies are needed to set up and we shall do justice to that in this article. Feel free to brush up concepts of Django that require mastering in this article [Core Concepts of Django to master](/https://section.io/engineering-education/core-concepts-of-django-to-learn/), though it is not a prequisite for this.
+
+#### Prequisites
+1. Latest version of Python is required. So head on to `Python Docs` and download if you are not having it locally.
+[Python Websites](/https://www.python.org/downloads/)
+
+2. Latest Version of PostgreSQL database is needed. Download here.
+[Download Postgres](/https://www.postgresql.org/download/)
+
+3. Download workbench of postgreSQL here too. 
+[Download Workbench](/https://www.pgadmin.org/download/)
+
+4. Download git bash too. 
+[Git Bash](/https://git-scm.com/download/win)
```

Comment:
Be brief here for instance

### Prerequisites

1. [Python](https://www.python.org/downloads)
2.

Reply:
I will very soon.